hat/assets/js/apps/Iaso/domains/forms/components/FormAttachments.tsx

```diff
@@ -0,0 +1,130 @@
+import { AttachmentModal } from './AttachmentModal';
+
+export const defaultSorted = [{ id: 'updated_at', desc: false }];
+
+type Props = {
+    formId: string;
+    params: FormParams;
+};
+
+const useGetColumns = (): Column[] => {
+    const { formatMessage } = useSafeIntl();
+    const { mutateAsync: deleteAttachment } = useDeleteAttachment();
+    return [
+        {
+            Header: formatMessage(MESSAGES.name),
+            accessor: 'name',
+        },
+        {
+            Header: 'MD5',
+            accessor: 'md5',
+        },
+        {
+            Header: formatMessage(MESSAGES.created_at),
+            accessor: 'created_at',
+            Cell: DateTimeCell,
+        },
+        {
+            Header: formatMessage(MESSAGES.updated_at),
+            accessor: 'updated_at',
+            Cell: DateTimeCell,
+        },
+        {
+            Header: formatMessage(MESSAGES.actions),
+            accessor: 'actions',
+            sortable: false,
+            Cell: (settings: any): ReactElement => {
+                return (
+                    <>
+                        <Link href={settings.row.original.file} download>
+                            <IconButton
+                                onClick={() => null}
+                                tooltipMessage={MESSAGES.download}
+                                overrideIcon={GetAppIcon}
+                            />
+                        </Link>
+                        <DeleteDialog
+                            titleMessage={{
+                                ...MESSAGES.deleteAttachment,
+                                values: {
+                                    attachmentName: settings.row.original.name,
+                                },
+                            }}
+                            message={{
+                                ...MESSAGES.deleteWarning,
+                                values: {
+                                    planningName: settings.row.original.name,
+                                },
+                            }}
+                            onConfirm={() =>
+                                deleteAttachment(settings.row.original.id)
+                            }
+                        />
+                    </>
+                );
+            },
+        },
+    ];
+};
+
+export const FormAttachments: FunctionComponent<Props> = ({
+    formId,
+    params,
+}) => {
+    const { data: attachments, isFetching: isFetchingAttachments } =
+        useGetAttachments(formId, params);
+
+    const { mutateAsync: upload, isLoading: isUploading } =
+        useUploadAttachment(formId);
+    const dispatch = useDispatch();
+    const columns = useGetColumns();
+    return (
+        <Box>
+            <Box
+                mb={2}
+                justifyContent="flex-end"
+                alignItems="center"
+                display="flex"
+            >
+                <AttachmentModal
+                    iconProps={{}}
```

Comment:
Is that TS complaining if you leave it undefined? If so we should make the change in blsq-comp, no? (It's not exactly pressing though)